api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -41,9 +41,6 @@ default Map<String, DbUser> getUsersMappedByUsernames(Collection<String> usernam
   @Query("SELECT user FROM DbUser user")
   List<DbUser> findUsers();
 
-  @Query("SELECT user FROM DbUser user WHERE user.disabled = FALSE")
-  List<DbUser> findUsersExcludingDisabled();
```

Comment:
unused.  there is also an equivalent method `findUsersByUsernameInAndDisabledFalse` which duplicates this